config/inovelli/lzw60.xml

```diff
@@ -0,0 +1,131 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/031E:000d:0001</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/inovelli/lzw31-sn.png</MetaDataItem>
+    <MetaDataItem id="0001" name="ZWProductPage" type="0001">https://products.z-wavealliance.org/products/3723</MetaDataItem>
```

Comment:
ID/Type don't match the database entry

Reply:
ID and Type should be fixed now as well! Thank you.